draft-ietf-quic-transport.md

```diff
@@ -808,27 +809,27 @@ direction until that direction enters a terminal state, or until one of the
 endpoints sends CONNECTION_CLOSE.
 
 
-## Stream Final Offset {#final-offset}
+## Stream Final Size {#final-size}
 
-The final offset is the count of the number of bytes that are transmitted on a
-stream.  For a stream that is reset, the final offset is carried explicitly in a
-RESET_STREAM frame.  Otherwise, the final offset is the offset of the end of the
-data carried in a STREAM frame marked with a FIN flag, or 0 in the case of
-incoming unidirectional streams.
+The final size is the amount of data that is transmitted on a stream.  For a
+stream that is reset, the final size is carried explicitly in a RESET_STREAM
+frame.  Otherwise, the final size is the offset plus the length of a STREAM
+frame marked with a FIN flag, or 0 in the case of incoming unidirectional
+streams.
```

Comment:
I think that the current draft is incorrectly referring to incoming unidirectional streams; see #2286.